src/main/java/lotto/vo/LottoResultCount.java

```diff
@@ -0,0 +1,34 @@
+package lotto.vo;
+
+public class LottoResultCount {
+
+    private static final int INIT_COUNT = 0;
+    private static final int INCREASE_COUNT_UNIT = 1;
+
+    private final int count;
+
+    public LottoResultCount() {
+        this.count = INIT_COUNT;
+    }
+
+    public LottoResultCount(int count) {
+        this.count = count;
+    }
+
+    public int increase() {
+        return count + INCREASE_COUNT_UNIT;
```

Comment:
`LottoResultCount`를 VALUE OBJECT로 관리하면 어떨까요?

```suggestion
        return new LottoResultCount(count + INCREASE_COUNT_UNIT);
```